Automattic/wp-calypso
# composite-checkout: Account for product volume when highlighting discounts

#### Changes proposed in this Pull Request

* Depends on D44319-code, which exposes the "original subtotal" for each line item in the cart so we can show the correct original price for products with volume (e.g. gsuite).

#### Testing instructions

* Enter checkout with multiple gsuite licenses, as well as the discounted price. (Unfortunately I have not figured out how to trigger this.) You should see the correct pre-discount price in the gsuite line item.
